### DISCUSSION TITLE
Pyinstaller rocks

### DISCUSSION THREAD

**Discussion by gen2brain** [pyinstaller/pyinstaller#8859] [Features & Development]
I used it a long time ago in the project, and now for the new release, I was amazed by some of the new features. I didn't follow the development so this was new to me, but the `_internal` directory is a big hit. And, another one, I used to build separate Windows builds for GUI and the console. Now, I just built for console and `hide early` console, it is super nice, the same as in Linux, if you start it terminal you will see the output else you don't, perfect! This was just meant as a motivational boost, thank you for all the work!

**Comment by bwoodsend** [comment]
You're welcome. Although you probably want to read https://github.com/pyinstaller/pyinstaller/issues/8022 before you get too excited about `hide early` (I thought we'd documented its unfortunate caveat but it looks like we only got as far as _intending_ to document it).

**Reply by rokm** [reply]
We did document it as part of #8798.
